### PR TITLE
Added srcset and higher resolution image support

### PR DESCRIPTION
**Pull request by elkcl**
1. Simplebox now transfers the value of the srcset attribute.
2. I also added the ability to define postfixes. For example, you have an image called `panda.jpg` and a high-res version `panda_full.jpg`. You can define `postfix: "_full"` and now simplebox will seek a version with the `_full` postfix for every image with an additional class `hq`.

PS I tested it in Chrome and it worked, but I didn't test it in other browsers yet. Also I didn't update `simplebox.min.js` yet, because I don't know which minifier do you use.

### CODE REVIEW

**Comment by libhide** [comment]
Hey, thanks for this. I'll have a look at it over the weekend and get back to you with a more detailed response!

File: src/js/simplebox.js [hunk 1]
```diff
@@ -58,7 +59,10 @@
 
                 var $this = $(this);
                 var imageSRC = $this.attr("src");
-                if (~$this.attr("class").indexOf("hq")) imageSRC = imageSRC.substring(0, imageSRC.lastIndexOf(".")) + settings.postfix + imageSRC.substring(imageSRC.lastIndexOf("."));
+                if (~$this.attr("class").indexOf(settings.hqClass)) {
+                    let dotIndex = imageSRC.lastIndexOf(".");
```
Comment (libhide) [inline src/js/simplebox.js]: Hey, could you switch the `let` to a `var` for now? The project isn't in ES6 right now and using `let` just messes with `gulp`.

**Review by libhide — CHANGES_REQUESTED** [review]
Hey, thanks for the PR! I had a few thoughts:

1. Do you think telling the user to put a new class on the `` to support hq images is the way to go? I don't have an opinion on this so we can discuss this maybe. What was your rationale for using the new class?
2. Could you also please also update the docs (`README.md`)?

File: src/js/simplebox.js (hunk 1 above)
Comment (elkcl) [reply src/js/simplebox.js]: 1. Yes, I do. Basically, if someone wanted to use this new feature and had a lot of images with simplebox, they could enable it for 
only _some_ of the images, and this way other ones (that don't have a high-res version yet) won't break.
2. Yes, I'll do it a bit later.
3. Ok, I'll change it to `var`.

**Review by libhide — APPROVED** [review]

**Comment by libhide** [comment]
Hey, just one last thing before I can merge – could you also update the `README`'s advanced section with an example using `srcset`? I think the high res image support is clear now but there isn't any direction regarding setting up the `img` tags properly.

PS. the high res images work even if I don't specify an srcset – this is expected behaviour right?

**Comment by elkcl** [comment]
Yes, it's expected behaivour. `srcset` is just another attribute used to define multiple paths for different screen resolutions. I just thought at first that it could be used to implement high-res images support, but it didn't work, therefore I made the postfix system instead. `srcset` support is still implemented though if someone wanted to use it.

PS I read about `srcset` [here](https://developers.google.com/style/images)

**Pull request merged**